events/templates/events/create_event.html

```diff
@@ -0,0 +1,51 @@
+{% extends "base.html" %}
+    {% block head-content %}
+        {% load static %}
+        <link rel="stylesheet" type="text/css" href="{% static 'events/create_event.css' %}">
+        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.0-2/css/all.min.css">
+        <script src="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/4.5.0/js/bootstrap.min.js" type="text/javascript"></script>
+        <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.5.1/jquery.min.js" type="text/javascript"></script>
+    {% endblock %}
+    {% block body-content %}
+
+        <div class="container-shadow"></div>
+        <div class="container">
+            <div class="wrap">
+                <div class="headings">
+                    <a id="create-event" class="active"><span>Create Event:</span></a>
+                    {% if form.errors %}
+                    <div class="alert alert-danger">
+                        {{ form.errors }}
+                    </div>
+                    {% endif %}
+                </div>
+
+                <div id="create-event-form">
+                    <form method="POST">
+                        {% csrf_token %}
+
+                        <label id="title">Title <span class="required">*</span></label>
+                        <input id="input-title" type="text" name="title" value="{{form.title.value}}" required/>
+
+                        <label>Location </label>
+                        <input id="location" type="text" name="location" value="{{form.location.value}}" class="field-long"/>
+
+                        <label>Description </label>
+                        <textarea id="description" name="description" class="field-long field-textarea">{{form.description.value}}</textarea>
+
+                        <label>Date time start <span class="required">*</span></label>
+                        <input type="datetime-local" name="date_time_start" value="{{form.date_time_start.value}}" required/>
+
+                        <label>Date time end <span class="required">*</span></label>
+                        <input type="datetime-local" name="date_time_end" value="{{form.date_time_end.value}}" required/>
+
+                        <label>Color <span class="required">*</span></label>
+                        {{form.color}}
+                        <br>
```

Comment:
If you will use the widgets, so use can get rid of this lines and use only
```suggestion
{{form.as_p}}
```

Reply:
done 😄